contracts/CO30_Rob_eineMehrheitsbeteiligung.stubbhult/edit_me/briefing/situation.html

```diff
@@ -0,0 +1,27 @@
+<font color='#e3d310'>EMPLOYER</font>
+Name: Folkrepubliken Svea
+Location: Saxhult flygfält
+
+The national government has been fighting this insurgency for years now and the public is demanding a swift end to this bitter conflict in their coutnries history. 
+
+<font color='#d81717'>ENEMY FORCES</font>
+Name: Ättens Väktare
+Location: On and around Stora Färgen
+
+Ättens Väktare has found itself a welcome home to many of Europe's radicals, specifically those with white nationlist leanings. Many of their ranks have prior military experience
+in the nations of Europe, primarily the Bundeswehr, Bundesheer, Forsvaret, and Wojsko Polskie. With the proceeds from their drug trafficking, they have been able to arm them selves with 
+modern weaponry and some light vehicles. They have also recently purchased a small wing of rotary craft, some of which may be armed. Synixe can expect the majority of the groups forces to be centered
+around the area's ports and airfields. 
```

Comment:
```suggestion
Ättens Väktare has found itself a welcome home to many of Europe's radicals, specifically those with white nationalist leanings. Many of their ranks have prior military experience in the nations of Europe, primarily the Bundeswehr, Bundesheer, Forsvaret, and Wojsko Polskie. With the proceeds from their drug trafficking, they have been able to arm them selves with modern weaponry and some light vehicles. They have also recently purchased a small wing of rotary craft, some of which may be armed. Synixe can expect the majority of the groups forces to be centered around the area's ports and airfields. 
```
Same here, must be one line